_data/jobs.yml

```
@@ -1,3 +1,5 @@
+- {expires: 2020-09-14, location: 'Visual Geometry Group (VGG), Department of Engineering Science, University of Oxford, UK',
+    name: Research Software Engineering, url: 'https://my.corehr.com/pls/uoxrecruit/erq_jobspec_version_4.display_form?p_company=10&p_internal_external=E&p_display_in_irish=N&p_process_type=&p_applicant_no=&p_form_profile_detail=&p_display_apply_ind=Y&p_refresh_search=Y&p_recruitment_id=147314'}
```

For the title I think you just want Research Software Engineer?
```suggestion
    name: Research Software Engineer, url: 'https://my.corehr.com/pls/uoxrecruit/erq_jobspec_version_4.display_form?p_company=10&p_internal_external=E&p_display_in_irish=N&p_process_type=&p_applicant_no=&p_form_profile_detail=&p_display_apply_ind=Y&p_refresh_search=Y&p_recruitment_id=147314'}
```